vernac/record.ml

```diff
@@ -477,7 +477,7 @@ let check_template ~template ~poly ~univs ~params { Data.id; rdata = { DataR.min
   match template with
   | Some template, _ ->
     (* templateness explicitly requested *)
-    if poly && template then user_err Pp.(strbrk "template and polymorphism not compatible");
+    if poly && template then user_err Pp.(strbrk "template and polymorphism not compatible.");
```

Comment:
```suggestion
    if poly && template then user_err Pp.(strbrk "Template and polymorphism not compatible.");
```

Reply:
I thought `template` had here to be considered a "keyword", thus keeping its case. So maybe "Attributes template and polymorphism are not compatible."?

Reply:
That looks better, we could also put them in quotes.

Reply:
Ok, put in quotes. Done